dev/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM debian:sid
+
+ENV MEDIAWIKI_VERSION wmf/1.30.0-wmf.4
```

Comment:
There isn't really a reason to require a specific version in the image.
You can already take any of the current mediawiki images, mount your local copy of mediawiki (and extensions) over it and use it as a basic development environment.